libs/perun/pipes/src/lib/perun-pipes.module.ts

```diff
@@ -20,7 +22,10 @@ import { GroupSyncIconColorPipe } from './group-sync-icon-color.pipe';
     CustomTranslatePipe,
     GroupSyncIconPipe,
     GroupSyncToolTipPipe,
-    GroupSyncIconColorPipe
+    GroupSyncIconColorPipe,
+    UserFullNamePipe,
```

Comment:
you declared it in your PR for the second time
also you exported this pipe for second time